src/uct/api/uct.h

```diff
@@ -1764,12 +1766,13 @@ UCT_INLINE_API ucs_status_t uct_ep_fence(uct_ep_h ep, unsigned flags)
  * @ingroup UCT_TAG
  * @brief Short eager tagged-send operation.
  *
- * This routine sends a message using short eager protocol. Eager protocol
- * means that the whole data is sent immediately to the peer. The data is
- * provided as buffer and its length, and must not be larger than the
- * corresponding @a max_short value in @ref uct_iface_attr. The immediate
- * value delivered to the receiver is implicitly equal to 0. If it’s required
- * to pass non-zero imm value, @ref uct_ep_tag_eager_bcopy should be used.
+ * This routine sends a message using @ref uct_short_protocol_desc "short"
+ * eager protocol. Eager protocol means that the whole data is sent immediately
```

Comment:
I agree.  We should be more careful about the use of the terms "immediate", "immediately", "imm".  They can have multiple, conflicting, meanings.  In this context, I think it is okay to use "immediately" if it is further clarified as to the meaning.  For example, how about this:
  Eager protocol means that the data can be immediately sent to the receiver without any preceding communication, or handshake, to setup the transfer.